projects/kubernetes-sigs/cluster-api-provider-vsphere/build/create_manifests.sh

```diff
@@ -30,6 +30,9 @@ source "${MAKE_ROOT}/../../../build/lib/common.sh"
 
 cd $REPO
 
+sed -i '' 's/^\( *\) - --enable-leader-election$/&\
```

Comment:
`sed -i ''`
This syntax of `sed` inplace will work only on MacOS. Using `gsed` instead would allow us to use the same syntax as Nix systems.

```suggestion
SED=sed
if [ "$(uname -s)" = "Darwin" ]; then
    SED=gsed
fi
$SED -i '' 's/^\( *\) - "--leader-elect"$/&\
```

Same below